Makefile

```diff
@@ -85,10 +89,13 @@ install: clean_socket egg_info
 egg_info:
 	$(PYTHON) setup.py egg_info
 
-release: clean egg_info README
-	@echo "dnf install python3-wheel python3-twine"
+packages: egg_info README
 	$(PYTHON) setup.py packages
 	cd dist && for F in *.gz; do sha512sum $${F} > $${F}.sha512sum.txt; done
+
+release: clean
+	@echo "dnf install python3-wheel python3-twine"
+	$(MAKE) packages
```

Comment:
This will cause release to only work on Fedora systems ?
Do we really want this ?